admin_manual/installation/source_installation.rst

```diff
@@ -221,56 +221,18 @@ On a machine running a pristine Ubuntu 16.04 LTS server, install the required an
 
 ::
 
-    apt-get install -y apache2 mariadb-server libapache2-mod-php7.0 \
-        openssl php-imagick php7.0-common php7.0-curl php7.0-gd \
-        php7.0-imap php7.0-intl php7.0-json php7.0-ldap php7.0-mbstring \
-        php7.0-mcrypt php7.0-mysql php7.0-pgsql php-smbclient php-ssh2 \
-        php7.0-sqlite3 php7.0-xml php7.0-zip
+    apt-get install -y apache2 mariadb-server libapache2-mod-php7.2 \
```

Comment:
I don't have an Ubuntu 16.04 system any more, but to get PHP 7.2 on that, I think you will have to install ppa:ondrej/php - Ubuntu 16.04 comes with some default PHP version. Did it come with PHP 7.0?

But the section heading could be changed to Ubuntu 18.04, which comes with PHP 7.2. And these commands should work.

Reply:
Anyone on Ubuntu 16.04 has to anyway, by the end of the year, either "manually" install some later PHP or upgrade to Ubuntu 18.04